app.js

```diff
@@ -43,6 +45,16 @@ const job = new CronJob(
     0,    //utcOffset
 );
 
+const dailyStatsJob = new CronJob(
+    '59 * * * * *',
```

Comment:
this was for testing purposes

@kajahno We were not able to make this work, do we need to do something else to run the CronJobs locally?

Reply:
I'll have a look!

Reply:
OK that cron is not complete. Without the parameter 'autoStart' and 'utcOffset' it's difficult to both control when to start it and predict when it's going to run. There's a cronjob just above, that you can use as an example.

By the way what that cron is saying is: "run every minute in the second 59", that might not be what we want.

Reply:
got it, I'll check it out, yeah, we removed the params bc we thought they were not needed. 
we wanted to run it every minute for testing, but it never worked, it might be due to those params you mentioned.